service-base/src/main/java/fi/nls/oskari/domain/map/MaplayerGroup.java

```diff
@@ -1,13 +1,24 @@
 package fi.nls.oskari.domain.map;
 
 import fi.nls.oskari.util.JSONHelper;
+
+import org.json.JSONArray;
 import org.json.JSONObject;
 
+import java.util.List;
 import java.util.Map;
 
 public class MaplayerGroup extends JSONLocalizedName {
 
 	private int id;
+	
```

Comment:
Empty lines here are making it more difficult to spot the fields. Using them for grouping purposes is fine but in this case they aren't helping.